odoo/models.py

```diff
@@ -3124,18 +3124,34 @@ def fields_get(self, allfields=None, attributes=None):
         has_access = functools.partial(self.check_access_rights, raise_exception=False)
         readonly = not (has_access('write') or has_access('create'))
 
+        translated_attrs = ('string', 'help')
+        translations = {}
+        if self.env.lang:
+            for key in translated_attrs:
+                translations[key] = getattr(self.env['ir.translation'], f'get_field_{key}')(self._name)
+
         res = {}
         for fname, field in self._fields.items():
             if allfields and fname not in allfields:
                 continue
             if field.groups and not self.env.su and not self.user_has_groups(field.groups):
                 continue
 
-            description = field.get_description(self.env)
+            description = {'type': field.type}
+            for attr, prop in field.description_attrs:
+                if attr in translated_attrs:
+                    continue
+                value = getattr(field, prop)
+                if callable(value):
+                    value = value(self.env)
+                if value is not None:
+                    description[attr] = value
```

Comment:
Damned.  I actually wanted to get rid of that magic `field.description_attrs`, and use a plain stupid method `get_description()` overridden in field classes.

Note that you retrieve _all_ field attributes, even you only need one.  Wouldn't it be better to pre-filter field attributes before retrieving them?